app/views/dashboards/_table_content.html.erb

```diff
@@ -11,8 +22,8 @@
         <th>Phone</th>
         <th>Name</th>
         <th>LMP</th>
-        <th>Status</th>
-        <th>Appt date</th>
+        <th<%= string_sort.html_safe %>>Status<%= autosort_span.html_safe %></th>
+        <th<%= string_sort.html_safe %>>Appt date<%= autosort_span.html_safe %></th>
```

Comment:
Can we change this to something like `<th<%= autosort_string_html_attr(local_assigns) %>>`?